content/_data/events/sf-jam-oct16.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "SF JAM"
+date: "2016-10-12T18:00:00"
```

Comment:
while I was doing this I thought the need to have a consistent naming scheme for JAM file w/in GitHub. Some JAMs use a number and others have titles for each of their meetups. For the purpose of keeping things simple I decided to use the following format for file names city-jam-monthyear (using last 2 digits to the sake or brevity).